examples/informer/README.md

```diff
@@ -0,0 +1,19 @@
+# Informer Example
+
+Informers provide a high-level API for creating custom controllers for Kubernetes resources.
+
+This particular example demonstrates:
+
+* How to write an Informer against a core resource type and handle to create/update/delete events
+
+## Running
+
+``` go
+# assumes you have a working kubeconfig, not required if operating in-cluster 
+go run *.go -kubeconfig=$HOME/.kube/config
+```
+
+## Use Cases
+
+* Capturing resource events for logging to external systems (eg. monitor non-Normal Events and publish metrics to a time series database)
+* Creating lifecycle controllers for ThirdPartyResources (eg. coordinate create/update/delete of an external datastore represented via a ThirdPartyResource type)
```

Comment:
ah i have been looking for such examples. Thanks a ton for initiating this. Couple questions, mostly naive :-)
- i am now seeing shared_informer stuff and changes related to that. Is this example already old now ? It might be helpful to at least put a comment in the doc saying how this differs from it 
- I have seen simple controller examples which dont use the framework. A word or too about that would be helpful as well or how this is better example